### DISCUSSION TITLE
Which tool is useful profiling axmol?

### DISCUSSION THREAD

**Discussion by aismann** [axmolengine/axmol#847] [Q&A] [answered]
'Optick' a C++ Profiler For Games => https://github.com/bombomby/optick is very useful.

see also #845

Another good free  stand-alone graphics debugging tool:
https://github.com/baldurk/renderdoc

**Comment by aismann** [comment, marked as answer]
'Optick' a C++ Profiler For Games => https://github.com/bombomby/optick is very useful.

**Reply by johnspeny** [reply]
@aismann do you have a sample project on how you added it to your project?

**Reply by aismann** [reply]
@johnspeny 
At the moment? No sorry.
Maybe I will use Optick when I have all stuff implemented on a new extension: ` DrawNodeExt.`
